--- a/lib/functions/cli/commands.sh
+++ b/lib/functions/cli/commands.sh
@@ -78,6 +78,19 @@ function armbian_register_commands() {
 		["armbian-desktop"]="artifact"
 
 		["undecided"]="undecided" # implemented in cli_undecided_pre_run and cli_undecided_run - relaunches either build or docker
+
+		# experimental stuff
+		# example: ./compile.sh armvuan-build BOARD=orangepi-r1 BRANCH=current RELEASE=daedalus
+		["armvuan-prebuild"]="armvuan"
+		["armvuan-stage1"]="armvuan"
+		["armvuan-stage2"]="armvuan"
+		["armvuan-add-armbian-repos"]="armvuan"
+		["armvuan-tweaks"]="armvuan"
+		["armvuan-blend"]="armvuan"
+		["armvuan-final-tweaks"]="armvuan"
+		["armvuan-make-image"]="armvuan"
+
+		["armvuan-build"]="armvuan"  # all the above in one go
 	)
 
 	# common for all CLI-based artifact shortcuts
